packages/core/admin/admin/src/content-manager/components/BlocksInput/hooks/useModifiersStore.tsx

```diff
@@ -59,7 +58,7 @@ type ModifiersStore = {
  * Manages a store of all the available modifiers.
  */
 function useModifiersStore(): ModifiersStore {
```

Comment:
do you think it's worth just scrapping this hook and moving it into your `useBlocksEditorContext` hook? at the moment the latter is just a shell re-exporting stuff, feels like a mental overhead we maybe don't need?

Reply:
@joshuaellis yes I also would love to get rid of it, but I've been running into issues. Something about my editor value being outdated, like a stale closure which I couldn't get around. I actually have it stashed somewhere if you're interested in doing PP on it

Reply:
Do you think this case of stale editor value could happen at other places as well we have not yet encountered?

Reply:
@madhurisandbhor I don't think so, we have lots of tests, and with this current approach the modifiers object has direct access to the editor via useSlate so it's should never be stale. That's pretty much how it's been working until now.

Reply:
> I actually have it stashed somewhere if you're interested in doing PP on it

Yep, just find any slot in my calendar that works for you and we can work through it!